scripts/FetchFromYouTube.java

```diff
@@ -0,0 +1,131 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
```

Comment:
What about adding a readme in this folder to explain how/when to use these scripts?

Reply:
Fair point, I will add it in a separate PR